.github/workflows/python-integration.yml

```diff
@@ -0,0 +1,43 @@
+name: Python integration tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  integration:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.8, 3.9, '3.10']
+
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
What v2 are we talking here?

Reply:
The action is defined here: https://github.com/actions/checkout

I'm using v2 because that's what I used in the past, but I just noticed there's now a v3.